libs/sdk-ui-theme-provider/src/ThemeProvider/Context.tsx

```diff
@@ -0,0 +1,91 @@
+// (C) 2019 GoodData Corporation
+import React from "react";
+import compose from "lodash/flowRight";
+import { ITheme } from "@gooddata/sdk-model";
+import { wrapDisplayName } from "@gooddata/sdk-ui";
+
+const ThemeContext = React.createContext<ITheme | undefined>(undefined);
+ThemeContext.displayName = "ThemeContext";
+
+const ThemeIsLoadingContext = React.createContext<boolean | undefined>(undefined);
+ThemeIsLoadingContext.displayName = "ThemeIsLoadingContext";
+
+/**
+ * @beta
```

Comment:
please add TSDoc for all things marked as beta. again, look at it from library consumer perspective. for functions / component, describe what is the component good for.

for props interfaces, no need to describe the interface itself, but please describe the different properties